lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebarFooter.test.tsx

```diff
@@ -4,63 +4,61 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
-
-import NxGlobalSidebarFooter, { NxGlobalSidebarFooterProps as Props }
-  from '../NxGlobalSidebarFooter';
-
-import NxTextLink from '../../NxTextLink/NxTextLink';
+import NxGlobalSidebarFooter, { NxGlobalSidebarFooterProps as Props } from '../NxGlobalSidebarFooter';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 describe('NxGlobalSidebarFooter', function() {
   const minimalProps: Props = {},
-      getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxGlobalSidebarFooter, minimalProps);
+      quickRender = rtlRender<Props>(NxGlobalSidebarFooter, minimalProps),
+      renderEl = rtlRenderElement<Props>(NxGlobalSidebarFooter, minimalProps);
+
+  it('passes additional classes specified as props to <footer>', function() {
+    const el = renderEl({ className: 'test-class' }),
+        defaultEl = renderEl()!;
+
+    expect(el).toHaveClass('test-class');
 
-  it('renders an <footer> with nx-global-sidebar__footer class and those passed as props', function() {
-    expect(getShallowComponent({ className: 'test-class' }))
-        .toMatchSelector('footer.nx-global-sidebar__footer.test-class');
+    for (const cls of Array.from(defaultEl.classList)) {
+      expect(el).toHaveClass(cls);
+    }
   });
 
   it('passes additional attributes to <footer>', function() {
-    expect(getShallowComponent({ id: 'test-id' })).toHaveProp('id', 'test-id');
-    expect(getShallowComponent({ lang: 'en-ca' })).toHaveProp('lang', 'en-ca');
+    expect(renderEl({ id: 'test-id' })).toHaveAttribute('id', 'test-id');
+    expect(renderEl({ lang: 'en-ca' })).toHaveAttribute('lang', 'en-ca');
   });
 
   it('renders the Support div with a link with the passed href and text and does not render when empty', function() {
```

Comment:
Instead of asserting that the div is in the DOM, perhaps the test could look at if the component renders link in a variety of situations: (if neither are provided, if the supportLink is not provided but the supportText is,  if the supportLink is provided and support text isn't, and yes if both are provided). And then still assert the href attribute and that the text is correct.

Reply:
> Instead of asserting that the div is in the DOM, perhaps the test could look at if the component renders link in a variety of situations: (if neither are provided, if the supportLink is not provided but the supportText is, if the supportLink is provided and support text isn't, and yes if both are provided). And then still assert the href attribute and that the text is correct.

I've modified the tests (also incudes your suggestions) here: 4d234960d5a980fe8bfd11ddb392c5085d58bb8c

Reply:
Looks good!